conda-forge.yml

```
@@ -1,6 +1,9 @@
-github:
-  branch_name: main
-  tooling_branch_name: main
 conda_build:
```

Would use cross-compilation if possible

If this doesn't work, we may need to [enable Travis CI on the feedstock]( https://github.com/conda-forge/admin-requests?tab=readme-ov-file#request--revoke-access-to-ci-resources ) to run native builds

```suggestion
build_platform:
  linux_aarch64: linux_64
  linux_ppc64le: linux_64
conda_build:
```